libfreeipmi/cmds/ipmi-lan-cmds.c

```diff
@@ -756,6 +756,366 @@ fiid_template_t tmpl_cmd_get_lan_configuration_parameters_bad_password_threshold
     { 0, "", 0}
   };
 
+fiid_template_t tmpl_cmd_get_lan_configuration_parameters_ipv6_ipv4_support_rs =
```

Comment:
I think you need to take a look at the IPMI spec 23.2.  All Get Lan Conf responses begin with

```
    { 8, "cmd", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
    { 8, "comp_code", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED | FIID_FIELD_MAKES_PACKET_SUFFICIENT},
    { 4, "present_revision", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
    { 4, "oldest_revision_parameter", FIID_FIELD_REQUIRED | FIID_FIELD_LENGTH_FIXED},
```

before all of the table information information.  Sorry, I was saying "generically" that the packets appeared to be wrong in the prior review.  There shouldn't be a "channel number" or "parameter selector" anywhere.